xgi/classes/hypergraph.py

```diff
@@ -512,10 +513,20 @@ def add_edges_from(self, ebunch_to_add, **attr):
 
         Parameters
         ----------
-        ebunch_to_add : iterable of edges
-            Each edge given in the iterable will be added to the
-            graph. Each edge must be given as as an iterable of nodes
-            or an iterable with the last entry as a dictionary.
+        ebunch_to_add : iterable
+
+            An iterable of edges, in one of the following formats.
+
+            * Format 1: Iterable of hashables (id1, id2, ...), or
```

Comment:
It's more important to know it's node ids (which I didn't find clear at first), rather than that it must be hashable. Also this (id1, id2, ...) is the same as `members` in the other formats I think. Maybe we can make those things more explicit?

Maybe here just 
"* Format 1: Iterable of hashables nodes IDs (id1, id2, ...), or

Reply:
Good catch, I updated the docs.